client/my-sites/plan-features-2023-grid/plan-comparison-grid.tsx

```diff
@@ -1,21 +1,193 @@
+import {
+	applyTestFiltersToPlansList,
+	FeatureGroup,
+	getPlanFeaturesGrouped,
+	PLAN_ENTERPRISE_GRID_WPCOM,
+} from '@automattic/calypso-products';
+import { Gridicon } from '@automattic/components';
 import styled from '@emotion/styled';
 import { useTranslate } from 'i18n-calypso';
+import { useMemo } from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { getPlanFeaturesObject } from 'calypso/lib/plans/features-list';
+import { IntervalTypeToggle } from 'calypso/my-sites/plans-features-main/plan-type-selector';
 import { PlanProperties } from './types';
+
 const PlanComparisonHeader = styled.h1`
 	font-size: 2rem;
 	text-align: center;
 	margin: 48px 0;
 `;
 
-type PlanComparison2023GridProps = { planProperties?: Array< PlanProperties > };
+const Title = styled.div`
+	font-weight: 500;
+	font-size: 20px;
+	margin-bottom: 15px;
+`;
+
+const Grid = styled.div`
+	display: grid;
+`;
+const Row = styled.div`
+	display: flex;
+	justify-content: space-between;
+	padding: 14px 0px;
+	border-bottom: 1px solid #eee;
+	.plan-comparison-grid__feature-feature-name {
+		width: 250px;
+	}
+`;
 
-export const PlanComparison2023Grid: React.FC< PlanComparison2023GridProps > = () => {
+const Cell = styled.div< { textAlign?: string } >`
+	text-align: ${ ( props ) => props.textAlign ?? 'left' };
+`;
+
+type FeatureAvailabilityForPlansProps = {
+	displayedPlans?: Array< PlanProperties >;
+	featureSlug: string;
+	planFeatureMap: Record< string, Set< string > >;
+};
+
+const FeatureAvailabilityForPlans: React.FC< FeatureAvailabilityForPlansProps > = ( {
+	displayedPlans,
+	featureSlug,
+	planFeatureMap,
+} ) => {
+	return (
+		<>
+			{ ( displayedPlans ?? [] ).map( ( { planName } ) => {
+				if ( planFeatureMap[ planName ].has( featureSlug ) ) {
+					return (
+						<Cell
+							key={ planName }
+							className={ `plan-comparison-grid__plan ${ planName }` }
+							textAlign="center"
+						>
+							<Gridicon icon="checkmark" color="#0675C4" />
+						</Cell>
+					);
+				}
+				return (
+					<Cell
+						key={ planName }
+						className={ `plan-comparison-grid__plan ${ planName }` }
+						textAlign="center"
+					>
+						<Gridicon icon="minus-small" />
+					</Cell>
+				);
```

Comment:
I'm wondering if this can be reduced to:
```jsx
<Cell
	key={ planName }
	className={ `plan-comparison-grid__plan ${ planName }` }
	textAlign="center"
>
{ planFeatureMap[ planName ].has( featureSlug ) ? <Gridicon icon="checkmark" color="#0675C4" /> : <Gridicon icon="minus-small" />  }
</Cell>
```